### DISCUSSION TITLE
How to add additional ordered list RTE with alphabets and Roman Numbers

### DISCUSSION THREAD

**Discussion by mikeljoe1** [react-page/react-page#1145] [Q&A] [answered]
Is it possible to add extra ordered list with alphabets/roman numbers in RTE ?

**Comment by macrozone** [comment, marked as answer]
of course if all lists should be alphabets, you could simply use css:

```
.react-page ol {
    list-style-type: lower-alpha;
}
```

but i guess you want to have an extra list plugin next to the numbered list?

This was already possible, but i realized its a bit awkward to do, so i pushed an update (4.5.0).

There you also find an example: https://react-page.github.io/examples/customListInSlate

source: https://github.com/react-page/react-page/blob/master/examples/pages/examples/customListInSlate.tsx#L14

**Reply by mikeljoe1** [reply]
@macrozone  Thanks for checking this. I will try this and let you know
